controller/test-files/work_item_link/list/ok.res.paylpad.golden.json

```diff
@@ -87,11 +87,22 @@
         "system.description.rendered": "\u003cp\u003e\u003ccode\u003e(see function github.com/fabric8-services/fabric8-wit/controller_test.(*workItemLinkSuite).TestList in controller/work_item_link_blackbox_test.go)\u003c/code\u003e\u003c/p\u003e\n",
         "system.metastate": null,
         "system.number": 1,
-        "system.order": 1,
```

Comment:
how is the `system.order` 1 here

Reply:
`1` looks wrong to me. I don't know why it was 1